### ISSUE TITLE
info-all = 'false' attribute not working

### ISSUE THREAD

**Issue by BasvdM** [frapontillo/angular-bootstrap-duallistbox#13]
Hi! 

Thanks for creating this angular version of the duallistbox.

I tried to use the  info-all = 'false' setting, but instead of hiding the info text it shows the text 'false' above the dual list box.

I changed the code myself by adding an extra transformFn to the InfoAll attribute.

```
    var getFalseOrStringValue = function (attributeValue) {
        if (attributeValue === false || attributeValue === 'false') return false;
        return attributeValue;
    };
```

...and...

```
        'infoAll': {
          changeFn: 'setInfoText',
          defaultValue: 'Showing all {0}',
          transformFn: getFalseOrStringValue
        },
```

I am not keen on making own changes. If you could provide a fix I would be very gratefull.

**Comment by fabiogel** [comment]
Hello BasvdM! You can use a special character as '&shy.;'(without dot).

Its work for me. Cya

**Comment by BasvdM** [comment]
Hi,

Thanks for your reply. Unfortunately I don't understand your answer....

What should I do with the soft hyphen?

Gr. Bas

**Comment by fabiogel** [comment]
Wow, srry! Use info-all="&s.hy;"

Remove the dot in the especial character.

**Comment by frapontillo** [comment]
Can I see a live demo of the issue please?
